charts/e2g/templates/bundle.yaml

```diff
@@ -73,5 +73,11 @@ resources:
           {{ $e | nindent 10 }}
       {{- end -}}
     {{- end -}}
+    {{- if $.Values.e2g.filtergroups }}
+      {{- range $i, $e := $.Values.e2g.filtergroups }}
+        e2guardianf{{ .id }}.conf: |-
+          {{ .config | nindent 10 }}
+      {{- end -}}
+    {{- end -}}
```

Comment:
Don't you think it could become an issue having so many configurations in one Configmap? You might very quickly exceed the limit of 1MiB.

Reply:
yes i moved them to a separate configmap